scss/_settings.scss

```diff
@@ -91,44 +91,50 @@ $fd-colors: map-merge((
   "background": (
     1: hsl(210, 9%, 93.7%), // #EDEFF0 UI Shell/ App background
     2: hsl(210, 0%, 100%),  // #FFFFFF UI content background
-    3: hsl(145, 75%, 96.7%),  // #F1FDF6 Semantic Colors positive background
+    3: hsl(145, 10%, 97%),  // #E4FCEE Semantic Colors positive background
     4: hsl(28, 86.7%, 97.1%), // #FEF7F1 Semantic Colors alert background
     5: hsl(0, 100%, 96.1%),  // #FFEBEB Semantic Colors negative background
-    6: hsl(0, 100%, 94.7%), // #FFE4E4 Semantic Colors Neutral background
+    6: hsl(210, 4%, 98%), // #F5FAFF Semantic Colors Neutral background
+    7: hsl(0, 0, 96%) // #F4F4F4
   ),
   "neutral": (
     1: hsl(210, 3%, 98%), // #FAFAFA UI Neutral UI 1
     2: hsl(210, 3%, 93.5%), // #EEEEEF UI Neutral UI 2
     3: hsl(0, 0%, 85%), // #D9D9D9 UI Neutral UI 3
-    4: hsl(212, 8%, 57%) // #89919A UI Neutral UI 4
+    4: hsl(212, 8%, 57%), // #89919A UI Neutral UI 4
+    5: hsl(0, 0%, 97%), // #F7F7F7 UI Neutral UI 5
+    6: hsl(0, 0%, 95%) // #F2F2F2 UI Neutral UI 6
   ),
   "status": (
     1: hsl(147, 85.3%, 26.7%), // #107E3E Semantic Colors positive
     2: hsl(28, 90.2%, 48%), // #E9730C Semantic Colors alert
     3: hsl(0, 100%, 36.7%), // #BB0000 Semantic Colors negative
     4: hsl(210, 2.8%, 42.7%), // #6A6d70 Semantic Colors neutral text
-    5: hsl(210, 90.9%, 42.9%), // #6A6D70 Semantic Colors neutral border
+    5: hsl(210, 90.9%, 42.9%) // #6A6D70 Semantic Colors neutral border
   ),
   "accent": (
-    1: hsl(34, 82%, 49%), // #E38B16 accent 1
-    2: hsl(0, 80%, 67%), // #DC7474 accent 2
+    1: hsl(34, 90%, 61%), // #D08014 accent 1
+    2: hsl(0, 68%, 50%), // #D04343 accent 2
     3: hsl(332, 75%, 49%), // #DB1F77 accent 3
     4: hsl(315, 54%, 49%), // #C0399F accent 4
     5: hsl(238, 65%, 63%), // #6367DE accent 5
-    6: hsl(210, 64%, 60%), // #5899DA accent 6
-    7: hsl(186, 81%, 39%), // #13A4B4 accent 7
+    6: hsl(210, 78%, 45%), // #286EB4 accent 6
+    7: hsl(186, 90%, 49%), // #0F828F accent 7
     8: hsl(75, 86%, 34%), // #7CA10C accent 8
     9: hsl(269, 54%, 58%), // #925ACE accent 9
-    10: hsl(205, 15%, 58%), // #8497a4 accent 10
+    10: hsl(206, 26%, 49%), // #647887 accent 10
     11: hsl(34, 82%, 45%), // #D17F15 accessible accent 1 B (3:1)
     12: hsl(0, 60%, 54%), // #D04343 accessible accent 2 B (4.6:1)
     13: hsl(210, 64.2%, 47.1%), // #2B78C5 accessible accent 6 B (4.6:1)
     14: hsl(238, 65.1%, 62.9%), // #0F828F accessible accent 7 B (4.6:1)
-    15: hsl(274.2, 60%, 55.9%), // #647887 accessible accent 10 B (4.6:1)
+    15: hsl(274.2, 60%, 55.9%) // #647887 accessible accent 10 B (4.6:1)
   ),
   "shell": (
-    1: hsl(210, 28%, 29%), // #354A5F shell background
-    2: hsl(210, 100%, 91%) // #D1E8FF shell icons
+    1: hsl(210, 28%, 29%), // #354A5F
```

Comment:
those colors are not reflected in https://deploy-preview-1359--fundamental.netlify.com/foundation/colors.html